src/Umbraco.Tests.Integration/Umbraco.Infrastructure/Services/MemberTypeServiceTests.cs

```diff
@@ -23,7 +23,7 @@ public class MemberTypeServiceTests : UmbracoIntegrationTest
         public void SetupTest()
         {
             // TODO: remove this once IPublishedSnapShotService has been implemented with nucache.
-            Core.Services.Implement.MemberTypeService.ClearScopeEvents();
+            global::Umbraco.Core.Services.Implement.MemberTypeService.ClearScopeEvents();
```

Comment:
This ⬇️ would work if you do `private MemberTypeService MemberTypeService => (MemberTypeService) GetRequiredService<IMemberTypeService>();` at the beginning of the file

```suggestion
            MemberTypeService.ClearScopeEvents();
```

Reply:
See above.